src/Symfony/Component/BrowserKit/Client.php

```diff
@@ -367,7 +367,18 @@ public function request(string $method, string $uri, array $parameters = array()
             return $this->crawler = $this->followRedirect();
         }
 
-        return $this->crawler = $this->createCrawlerFromContent($this->internalRequest->getUri(), $this->internalResponse->getContent(), $this->internalResponse->getHeader('Content-Type'));
+        $this->crawler = $this->createCrawlerFromContent($this->internalRequest->getUri(), $this->internalResponse->getContent(), $this->internalResponse->getHeader('Content-Type'));
+
+        if ($this->followRedirects) {
+            // Check for meta refresh redirect.
+            $redirect = $this->hasMetaRefresh();
+            if ($redirect) {
```

Comment:
according to previous comment: `if ($this->followRedirects && null !== $redirect = $this->getMetaRefresh()) {`
(removing one indentation level)